packages/prbnmcn-stats/prbnmcn-stats.0.0.2/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Basic statistics"
+maintainer: ["igarnier@protonmail.com"]
+authors: ["Ilias Garnier"]
+license: "MIT"
+homepage: "http://github.com/igarnier/prbnmcn-stats"
+bug-reports: "http://github.com/igarnier/prbnmcn-stats"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.12.0"}
+  "qcheck" {>= "0.17" & < "0.2" & with-test}
```

Comment:
this makes it untestable: "0.2" < "0.17"
https://opam.ocaml.org/doc/Manual.html#version-ordering

Reply:
Ah, stupid me. Thanks, will fix